src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -617,11 +614,6 @@ private void RunInstallCommands(InstallationContext installationContext, IEnumer
                     }
                     catch (Exception exception)
                     {
-                        if (exception.MustBeRethrown())
-                        {
-                            throw;
-                        }
-
```

Comment:
Perhaps we should rethrow here immediately when the exception is severe / fatal.